Graphics/IRenderer.hpp

```diff
@@ -7,42 +7,102 @@
 
 #pragma once
 
-#include "IWindow.hpp"
+#include <string>
+#include "Text/IText.hpp"
+#include "Frame/IFrame.hpp"
+#include "Window/IWindow.hpp"
 
 namespace ArcadeShared::Graphics
-{
+{   
+        typedef enum {
```

Comment:
I don't understand, `Close` is an event but not `A` or `B`...
Like, alphabet is for an only `keyPressed` event?

Reply:
I based myself on the SFML system at this level, it has only one enum with all the events so I took the same is removed those who do not interest us for our arcade !